src/main/bower.json

```diff
@@ -10,11 +10,16 @@
     "angular-route": "^1.3.0",
     "angular-sanitize": "^1.3.0",
     "angular-touch": "^1.3.0",
-    "angular-ui-router": "ui-router#^0.2.18"
+    "angular-ui-router": "ui-router#^0.2.18",
+    "angular-bootstrap": "^1.3.2",
+    "ngDatepicker": "*"
```

Comment:
@Joamena it's kinda dangerous leave open the version of the package. Also noticed that you're requesting this dependency on the package.json as well, you should choose one and go with it.